### ISSUE TITLE
Listen reponse message

### ISSUE THREAD

**Issue by lasaa2** [ExistentialAudio/SwiftOSC#47]
Hi,

I have OSCserver (Behringer X32 digital sound mixer) that send replies to sender's ip & port. There is no settings to define reply ip & port in X32.

Is it possible to create listener (OSCserver?) for that with this framework? It would be nice to get response messages to IOS app.

- Lassi

**Comment by devinroth** [comment]
There have been a few people who have asked about this. 

I don’t have a x32 but from what I remember you need to send a broadcast message. And then the X32 responds to the outgoing port (which is set by the OS). I don’t think you’re able to set the outgoing port manually but I could be wrong. I just haven’t found a way. Personally I think the way the X32 deals with this is makes things complicated. 

In short it’s definitely possible to do what you want but there isn’t a quick and easy way since the x32 make it tricky.
